packages/starknet-snap/src/utils/din.test.ts

```diff
@@ -0,0 +1,23 @@
+import { constants } from 'starknet';
+
+import { getRPCUrl } from './din';
+
+describe('getRPCUrl', () => {
+  it('returns Mainnet RPC URL if chain id is Mainnet', () => {
+    expect(getRPCUrl(constants.StarknetChainId.SN_MAIN)).toBe(
+      'https://starknet-mainnet.infura.io/v3/',
```

Comment:
can we confirm the blastAPI din URL is the same with https://starknet-mainnet.infura.io/v3/